pkg/resource/service.go

```diff
@@ -65,20 +68,43 @@ func (r *ServiceResource) Schema(_ context.Context, _ resource.SchemaRequest, re
 				Description: "Password for the default user. One of either `password` or `password_hash` must be specified.",
 				Optional:    true,
 				Sensitive:   true,
+				Validators: []validator.String{
+					stringvalidator.ConflictsWith(path.Expressions{path.MatchRoot("double_sha1_password_hash")}...),
+					stringvalidator.AtLeastOneOf(path.Expressions{
+						path.MatchRoot("password_hash"),
+					}...),
+				},
 			},
 			"password_hash": schema.StringAttribute{
 				Description: "SHA256 hash of password for the default user. One of either `password` or `password_hash` must be specified.",
 				Optional:    true,
 				Sensitive:   true,
+				Validators: []validator.String{
```

Comment:
Maybe go step further and actually try to decode base64 instead of validating against regex? I guess it will also work faster. (not sure if validator.String can have some custom validator passed)

Reply:
Ah I see it was just moved from other place so it is not really related to PR.

Reply:
it's a good suggestion, I will keep it for a future round of improvements